### PR TITLE
[sui-execution][EASY] README

### PR DESCRIPTION
**Pull request by amnn**
## Description

Adding a brief readme to explain this internal tool.

## Test Plan

:eyes:

---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)

- [ ] protocol change
- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

**Review by stefan-mysten — APPROVED** [review]

**Review by rvantonder — APPROVED** [review]

File: sui-execution/cut/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+that `cut` would make without committing them to the filesystem.
+
+## Properties
+
+When making copies, `cut` maintains the following properties:
+
+- If crate A depends on crate B, then when copies A' and B' are made,
+  A' will depend on B' (not B).
+- If crate A (copied) depends on crate B (not copied) and refers to it
+  by a relative path, the path in copy A' is fixed up to continue
```
Comment (ronny-mysten) [inline sui-execution/cut/README.md:14]:
```suggestion
  by a relative path, the path in copy A' continues
```

File: sui-execution/cut/README.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  an optional `suffix` parameter that is used to strip a common suffix
+  from the names of crates found in this directory.
+- A list of crate names to look for -- only crates whose names are in
+  this list will be copied, and if a crate in this list is not
+  encountered, a warning will be issued.
+
+## Copying
+
+When copying crates, the tool preserves the relative path of the crate
+in its source directory. I.e. if crate A is found at path
```
Comment (ronny-mysten) [inline sui-execution/cut/README.md:37]:
```suggestion
in its source directory. For instance, if crate A is found at path
```

File: sui-execution/cut/README.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+## Properties
+
+When making copies, `cut` maintains the following properties:
+
+- If crate A depends on crate B, then when copies A' and B' are made,
+  A' will depend on B' (not B).
+- If crate A (copied) depends on crate B (not copied) and refers to it
+  by a relative path, the path in copy A' is fixed up to continue
+  pointing to B
```
Comment (ronny-mysten) [inline sui-execution/cut/README.md:15]:
```suggestion
  pointing to B.
```
:)

**Review by ronny-mysten — APPROVED** [review]
Couple of nits that can be ignored.

**Pull request merged**